### PR TITLE
feat(ci): Add Gofish support

### LINKED ISSUE #855: Add GoFish support
**Description**

Hello cosign community and @cpanato,

I added GoFish support via goreleasers (@caarlos0). 

Thanks

### PR DESCRIPTION
**Pull request by dirien**
Signed-off-by: Engin Diri <engin.diri@mail.schwarz>

#### Summary

Hello cosign community and @cpanato,

I added GoFish support via goreleasers (@caarlos0). The only things we need before merging this PR:

A github repo called: fish-food  similar to https://github.com/goreleaser/fish-food and and ensure access to it during the release process.

```yaml
owner: sigstore
name: fish-food
```

#### Ticket Link

Fixes #855 

#### Release Note

NONE
```release-note

```

### CODE REVIEW

**Comment by dirien** [comment]
Hi @caarlos0 and @cpanato, here the PR continues. I screwed the original one. :/

**Comment by dlorenc** [comment]
I think we have the repo setup here: https://github.com/sigstore/fish-food

Is there anything we have to do to get permissions setup correctly?

**Comment by cpanato** [comment]
i will test this this week in my fork to validate, bear with me

**Comment by cpanato** [comment]
so sorry @dirien for the huge delay on this

I'm finishing some things and will try this out soon, please bear with me 🐻

**Comment by cpanato** [comment]
did a test and the outcome can be found here: https://github.com/cpanato/fish-food-test

looks like we are publishing all, is that we want? or only cosign and sget?

**Comment by dirien** [comment]
rebase done, and added the newline

File: .goreleaser.yml [hunk 1]
```diff
@@ -213,3 +213,12 @@ release:
...
   extra_files:
     - glob: "./release/release-cosign.pub"
+
+rigs:
+  - rig:
+      owner: sigstore
+      name: fish-food
+    homepage: https://sigstore.dev
+    description: Container Signing, Verification and Storage in an OCI registry.
+    license: "Apache License 2.0"
```
Comment (cpanato) [inline .goreleaser.yml:223]: @dirien can you add the commit author to use the sigstore bot

```
commit_author:
  name: sigstore-bot
  email: 86837369+sigstore-bot@users.noreply.github.com
```

after that we are good to go

Comment (dirien) [reply .goreleaser.yml:226]: Done :)

**Review by cpanato — APPROVED** [review]
thanks for this 🎉  and sorry for taking too long :(

@dlorenc we need to make sure the bot is added in the fish-food repo as well

**Comment by dlorenc** [comment]
Should I merge? The bot looks added:

image

**Comment by cpanato** [comment]
@dlorenc we are good in this side

**Pull request merged**